test/test_stall.py

```diff
@@ -0,0 +1,31 @@
+from mpi4py import MPI
+import horovod.tensorflow as hvd
+import tensorflow as tf
+import time
+import os
+import signal
+from common import env
+
+def test():
+  signal.alarm(45)
+  with env(HOROVOD_STALL_CHECK_TIME_SECONDS="2",
+    HOROVOD_STALL_SHUTDOWN_TIME_SECONDS="5"):
+    config = tf.ConfigProto()
+    tf.enable_eager_execution(config=config)
```

Comment:
Let's add `config.gpu_options.allow_growth = True`, otherwise each process will try to use up all the GPU memory.

Reply:
Or, since we don't require TensorFlow specific functionality, we could make this a PyTorch test, and model it after `test_torch.py`, which uses `unittest.TestCase`

Reply:
Changed to using torch.